src/__tests__/renderToString.spec.tsx

```diff
@@ -36,6 +36,17 @@ describe("renderToString", () => {
     expect(`${jsx}`).toBe(`<say-as interpret-as="date">foo</say-as>`);
   });
 
+  test("should convert amazon tag", () => {
+    const AmazonComponents = () => (
+      <speak>
+      <amazon-effect name="whispered">effect</amazon-effect>
+      <amazon-emotion name="excited" intensity="low">emotion</amazon-emotion>
+      <amazon-domain name="news">domain</amazon-domain>
+      </speak>
+    )
+    expect(renderToString(<AmazonComponents />)).toBe(`<speak><amazon:effect name=\"whispered\">effect</amazon:effect><amazon:emotion name=\"excited\" intensity=\"low\">emotion</amazon:emotion><amazon:domain name=\"news\">domain</amazon:domain></speak>`)
```

Comment:
These lines will cause an eslint error by rules that point out wrong indentation, unnecessary escapes and unspecified return type of function component.

For the last case, you able to use the `FC` type to determine the type of the function component.
```ts
const AmazonComponents: FC = () => (
  ...
);
```

Please run `yarn lint`, and fix it.

Reply:
Thanks for a feedback!
I've fixed these Lint issues.

Regards.